app/pages/multisig/AccountTransactions/CreateTransferProposal/CreateTransferProposal.tsx

```diff
@@ -79,14 +82,20 @@ function CreateTransferProposal({
 }: Props): JSX.Element {
     const dispatch = useDispatch();
 
+    const consensusStatus = useConsensusStatus();
+    const allowMemo = consensusStatus && consensusStatus.protocolVersion > 1;
```

Comment:
Refactor into a shared method (as this is used in multiple places) calling something like `nodeSupportsMemo()`.